src/rust/graph-generator-lib/src/local.rs

```diff
@@ -0,0 +1,197 @@
+use crate::serialization::SubgraphSerializer;
```

Comment:
This file contains code that is specific to running grapl locally (or, more specifically, "not on aws"). It performs extra tasks like waiting for S3 and SQS to be available (via localstack/minio) and feeding events to the generator.